File: frontend/src/Types/meetingParticipant.ts
import type { Guid } from './api';

export interface MeetingParticipant {
  id: Guid;
  meetingId: Guid;
  personId: Guid;
}

export interface CreateMeetingParticipantRequest {
  meetingId: Guid;
  personId: Guid;
}

export interface UpdateMeetingParticipantRequest {
  meetingId: Guid;
  personId: Guid;
}

File: backend/src/ScrumLog.Api/Data/Entities/MeetingParticipant.cs
namespace ScrumLog.Api.Data.Entities;

public class MeetingParticipant
{
    public Guid Id { get; set; }

    public Guid MeetingId { get; set; }

    public Meeting Meeting { get; set; } = null!;

    public Guid PersonId { get; set; }

    public Person Person { get; set; } = null!;
}
